src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,35 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){} // 2 violations , '{' is not followed and preceded by whitespace.
+ *     public static void main(String[] args) {
+ *         if (foo) { // ok
+ *             // body
+ *         }
+ *         else{ // violation
+ *             // body
+ *         }
+ *
+ *         for (int i = 1; i &gt; 1; i++) {} // violation ,
+ *                                             // '{' is not followed by whitespace.
+ *
+ *         Runnable noop = () -&gt;{}; // 2 violations ,
+ *                                         // '{' is not followed and preceded by whitespace.
+ *         try {
+ *             // body
+ *         } catch (Exception e){} // 2 violations ,
+ *                                     // '{' is not followed and preceded by whitespace.
+ *
+ *         for (char item: {'t','e'}) { // ok
```

Comment:
`// ok, because ignoreEnhancedForColon is true by default`

Reply:
Done with the changes @romani